jirate/jira_cli.py

```diff
@@ -476,19 +505,40 @@ def create_from_template(args):
         raise e
 
     for issue in template['issues']:
+        reserved_fields = ['subtasks']
+        required_fields = ['summary']
+        translate_fields = {'issue_type': 'issuetype'}
```

Comment:
For schemas/templates -> `issuetype`, I agree, we should just change it and remove the translation.

For switching to only "allowed field names", we'd need to provide a different schema or add custom field defs to the schema we're using run-time / dynamically since custom field defs are inherently installation-specific.

Reply:
I'd say the latter should be a separate patch, but I will certainly fix the former here.

Reply:
Yeah I didn't mean we should validate that stuff in the schema. Just let the creation logic assume it's being fed jira-valid fields, without translating them. So we agree :D

That kind of validation would be sweet to have _eventually_ but it's a whole other can of worms.

Reply:
I think so. Anyway, I added a bunch of nasty stuff to get some kind of testing to test the interactions from template-to-issue-creation-args. They all seem to work now.